rfcs/README.md

```diff
@@ -2,14 +2,58 @@
 
 We're trying a new approach of checking in Fluent UI RFCs to the repo so that they're more persistent, easier to find, and we can use PR comments to discuss the proposals.
 
+## Why RFCs?
+
+The RFC process is intended to bring a few benefits:
+
+- More efficent and durable decision making
+  - **Efficent** as we can all engage and review on our own time, rather than locking X engineers in a room for hours
+  - **Durable** as we give more time for decisions to be reviewed, commented, and considered. Rather than making a decision
+    after being locked in a room for an hour with X other engineers.
+- A more inclusive review proces
+  - Supporting both quick and deep thinkers
+  - Allowing proposals and comments from contributors
+- Capturing decisions, allowing:
+  - Us to review recent decisions and build ontop of them
+  - Others to join the project more easily
+
+The React RFC Process sums it up nicely[1]
+
+> The RFC process is a great opportunity to get more eyeballs on your proposal before it becomes a part of a released version. Quite often, even proposals that seem "obvious" can be significantly improved once a wider group of interested people have a chance to weigh in.
+>
+> The RFC process can also be helpful to encourage discussions about a proposed feature as it is being designed, and incorporate important constraints into the design while it's easier to change, before the design has been fully implemented.
+
 ## RFC process
 
-### Creation
+### 1. Creation
+
+- Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+- Write the RFC
+- Submit a PR and label it with **Type: RFC**.
+
+### 2. Review
 
-Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+A key purpose of the review phase is to seek feedback to help improve your idea. You the author are ultimately accountable for the proposal and it's impact. Reviewers can, through comments, help you make the best decision you can. Throughout this review and process it is important that we strive to avoid both [Design by Committee](https://en.wikipedia.org/wiki/Design_by_committee) and [Diffusion of Responsibility](https://en.wikipedia.org/wiki/Diffusion_of_responsibility).
 
-Once you're done writing, make a PR and label it with **Type: RFC**.
+- Seek support and integrate feedback.
```

Comment:
One thing I wonder if we should recommend specifically/only for RFCs is to use single comments rather than reviews with multiple comments--even though that's opposite of what I'd normally recommend/what I often do. The reason is I've noticed with the long, complex comment threads that can arise on RFCs, when some of the comments came as part of a review, the threads become extremely hard to follow (thanks to github's UI being terrible). In some cases you can reasonably find/read all the threads in the files view, but that doesn't work if a thread has disappeared from files view due to new changes touching that line.

Reply:
I'm not sure I understand the issue fully. 

Is this saying that when people use the "Start a Review" feature then sometimes the comment threads get broken? I can't say I've seen this happen so far. 

I'm nervous to start baking in guidance here that is specifically working around a particular behavior of the tools. Though again i'm not sure the impact of this particular issue.

Reply:
Yes that's what I meant. It's more as a suggestion than required guidance, but it's based on my experience with comment threads on my RFCs that became really hard to follow when some or all of the comments came in as part of larger reviews.

Reply:
Also since github PR reviews are the tool we're planning to use for RFCs, and the review interface is known to be far less than ideal in some ways, I think it's worth mentioning suggested mitigations in official guidance. (though in general I agree about avoiding guidance specific to tools that aren't an inherently required part of a process)

Reply:
Got it. Let's hold off for now. If it becomes a significant issue or if we have the need for other tips we can add it in.